ui/app/helpers/filter-wildcard.js

```diff
@@ -1,6 +1,12 @@
 import { helper as buildHelper } from '@ember/component/helper';
 
 export function filterWildcard([string, array]) {
+  if (!string || !array) {
+    return;
+  }
+  if (!string.id && string) {
```

Comment:
This block feels potentially superfluous, since in line 10 we just want essentially `string` as a string anyway. Maybe update to `let stringId = string.id || string` or a ternary would probably be safer

Reply:
Refactored a little.  Let's review to make sure it's as clean as can be.